text/src/autogluon/text/automm/data/collator.py

```diff
@@ -0,0 +1,324 @@
+import math
+import warnings
+import numpy as np
+import torch
+
+__all__ = ["Stack", "Pad", "Tuple", "List", "Dict", "PadImage", "Zip"]
+
+
+def _pad_arrs_to_max_length(arrs, pad_axis, pad_val, round_to=None, max_length=None):
```

Comment:
Is this code used to pad multi-dimensional sequences like:
```
[torch.rand([2,3]), torch.rand([2,8]), torch.rand([2,5])]
```
If it's just for one-dimensional sequences, then why not use [`torch.nn.utils.rnn.pad_sequence`](https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pad_sequence.html)?

Reply:
Please add tests for this function

Reply:
It can pad multi-dimensional input. Unit tests added.

Reply:
We also use padding for images, which have multi-dimensions. torch.nn.utils.rnn.pad_sequence seems not suitable.